integration/update_compat_test.go

```diff
@@ -125,7 +125,7 @@ func testUpdateCompat(t *testing.T, testCases map[string]updateCompatTestCase) {
 								targetErr = UnsetRaw(t, targetErr)
 								compatErr = UnsetRaw(t, compatErr)
 
-								// Skip updates that could not be performed due to Tigris schema validation.
+								// Skip inserts that could not be performed due to Tigris schema validation.
```

Comment:
I think in this case, we still skip updates, not inserts (I guess it was copy-pasted after an attempt to rewrite something 😉)

Reply:
Yeah this part of code was rewritten due to linters though I needed to "rollback" for now and I probably appended the comment by accident, nice catch :)